### ISSUE TITLE
test regression in fileinfo_fmt

### ISSUE THREAD

**Issue by asomers** [bolcom/libunftp#485]
PR #470 introduced a test regression:
```
> cargo test -p unftp-sbe-fs --lib tests::fileinfo_fmt
...
running 1 test
test tests::fileinfo_fmt ... FAILED

failures:

---- tests::fileinfo_fmt stdout ----
thread 'tests::fileinfo_fmt' panicked at crates/unftp-sbe-fs/src/tests.rs:169:5:
assertion failed: `(left == right)`

Diff < left / right > :
<-rwxr-xr-x            1            0            0              5  Jan 01 1970 tmp
>-rwxr-xr-x            1            0            0              5 Jan 01 00:00 tmp
```

cc @hirevo

_Originally posted by @asomers in https://github.com/bolcom/libunftp/issues/470#issuecomment-1677901729_

**Comment by Hirevo** [comment]
This test failure (which isn't really a regression) seems to now be fixed in commit 27b009a.

**Comment by hannesdejager** [comment]
Fixed
